lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -213,6 +216,79 @@ createTriangularMatrix(OpBuilder &b, Location loc, ValueRange payloadArgs,
   return success();
 }
 
+template <typename OpT>
+Value createDivModePayload(OpBuilder &b, Location loc,
+                           const TypeConverter *converter,
+                           ValueRange payloadArgs, OpT op,
+                           ArrayRef<Value> operands) {
+  static_assert(std::is_same_v<OpT, AtenDivTensorModeOp> ||
+                    std::is_same_v<OpT, AtenDivScalarModeOp>,
+                "template type must be a tensor/scalar div mode");
+  typename OpT::Adaptor adaptor(operands);
+  Type dtype = converter->convertType(op.getType())
+                   .template cast<RankedTensorType>()
```

Comment:
Typically we do `cast<RankedTensorType>(...)` instead of using the templated member function. It avoids the extract `template` part.